www/js/components/pages/user/limits/form.jsx

```diff
@@ -144,22 +149,24 @@ var Form = React.createClass({
         req.send({
             limit: limit,
             original: this.props.initialLimit
-        }).end(function(err, res) {
+        }).end(function (err, res) {
                 if (res.ok) {
                     this.props.onSaved(res);
+                } else {
+                    this.setState({formError: res.body});
                 }
             }.bind(this));
     },
 
-    render: function() {
-        return <div className="provisioning-limits-form">
-                <div className="modal-header">
+    render: function () {
+        return <div className="panel user-roles-form">
+            <div className="panel-body">
+                <div className="panel-title">
                 { this.props.initialLimit ?
                     <h4 className="modal-title">Update Limit for {this.props.initialLimit.datacenter}</h4> :
```

Comment:
<h4 className="modal-title">{this.props.initialLimit ? ('Update Limit for ' + this.props.initialLimit.datacenter) : 'Add New Limit' }</h4>